FirstProject/new-evaluation.html

```diff
@@ -0,0 +1,536 @@
+<!doctype html>
+<html class="no-js" lang="">
+<head>
+    <link rel="stylesheet" href="css/main.css">
+    <link rel="stylesheet" href="img/">
+    <!-- Latest compiled and minified CSS -->
+    <!--<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">-->
+
+    <!--&lt;!&ndash; jQuery library &ndash;&gt;-->
+    <!--<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>-->
+
+    <!-- Latest compiled JavaScript -->
+    <!--<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>-->
+</head>
+
+<body>
+    <nav class="row">
+        <div class="col-md-12" style="border-bottom: 2px solid darkgrey; padding-top: 12px;">
+            <div class="col-md-4" style="text-align:center;">
+                <img style="float:left; width: 12%;" href="#" style="width: 40%;" src="img/logo-v2.png"/>
```

Comment:
What does `href="#"` mean? More specifically, what does it mean when appended to an `img` tag?